microsoft-dynamics-nav/3.0/modules/ROOT/pages/ms-dynamics-nav-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,122 @@
+= Upgrading and Migrating Microsoft Dynamics NAV Connector to Version 3.0 - Mule 4
+
+Upgrade Anypoint Connector for Microsoft Dynamics NAV Connector to version 3.x.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|2.x |3.x
+|===
+
+== Changes in This Release
+
+To support Microsoft Dynamics NAV Connector in CloudHub 2.0, the client must provide SOAP requests in raw XML format instead of using plain old java object.
+
+Follow these steps to send or receive SOAP requests in raw XML format:
+
+. Make sure you have the prerequisites, such as Mule 4.6.6 or later, and obtain the WSDL file.
+. Identify all Mule flows that use Microsoft Dynamics NAV SOAP operations.
+. For the request body, update the content type to application/xml and provide SOAP requests in raw XML format instead of using POJOs.
```

Comment:
Can you fix the formatting in this step? The "XML for sending SOAP requests directly" and "XML for using the DataWeave transform method to convert JSON to XML payload" are not rendering like plaintext